jetty-server/src/main/java/org/eclipse/jetty/server/ServerConnector.java

```diff
@@ -281,7 +287,7 @@ public void setInheritChannel(boolean inheritChannel)
      * This open method can be called before starting the connector to pass it a ServerSocketChannel
      * that will be used instead of one returned from {@link #openAcceptChannel()}
      * @param acceptChannel the channel to use
-     * @throws IOException
+     * @throws IOException if the server channel cannot be bound
```

Comment:
IOException is throws if the channel IS NOT bound rather than cannot be bound.  This method is to pass in a pre-bound serverSocketChannel